libs/features/mas/src/aem-fragment.js

```diff
@@ -17,29 +18,24 @@ const fail = (message) => {
 
 /**
  * Get fragment by ID
- * @param {string} baseUrl the aem base url
  * @param {string} id fragment id
- * @param {string} author should the fragment be fetched from author endpoint
- * @param {Object} headers optional request headers
+ * @param {string} masCommerceService settings provider
  * @returns {Promise<Object>} the raw fragment item
  */
-export async function getFragmentById(baseUrl, id, author, headers) {
-    const endpoint = author
-        ? `${baseUrl}/adobe/sites/cf/fragments/${id}`
-        : `${baseUrl}/adobe/sites/fragments/${id}`;
+export async function getFragmentById(id, masCommerceService) {
+    const { env, wcsApiKey, locale } = masCommerceService.settings;
+    const host = env === 'prod' ? 'https://www.adobe.com' : 'https://www.stage.adobe.com';
+    const endpoint = `${host}/mas/io/fragment?id=${id}&api_key=${wcsApiKey}&locale=${locale}`;
```

Comment:
please use `https://mas.adobe.com/io/fragment` for preview

Reply:
discussed offline, for now will use www.stage.adobe.com/mas/io till we decide on preview endpoint